frontend/src/metabase/components/PaginationControls.jsx

```diff
@@ -54,7 +55,7 @@ export default function PaginationControls({
   );
 }
 
-const PaginationButton = IconWrapper.withComponent("button").extend`
+const PaginationButton = styled(IconWrapper.withComponent("button"))`
```

Comment:
You may be doing it very piecemeal, which is nice.

Just as note, `withComponent` [became a deprecation candidate in v4](https://styled-components.com/docs/api#withcomponent)

So you can write it like this:

https://github.com/metabase/metabase/pull/17001/files#diff-ed291e6d7be489c87c8767667547885cc55e808531f7f8c516cfad1f26b3a939R58-R61

Reply:
Let's do this once we get to v5.